Dockerfile

```diff
@@ -2,6 +2,10 @@ FROM nfcore/base
 LABEL authors="phil.ewels@scilifelab.se" \
       description="Docker image containing all requirements for the nfcore/chipseq pipeline"
 
+## GAWK has the 'and' function, needed for run_spp.R script from phantompeakqualtools package
+RUN apt-get update && apt-get install -y gawk && apt-get clean -y
+RUN echo 'alias awk=gawk' >> ~/.bashrc
```

Comment:
Couldn't we use the recipe from here instead? 

https://anaconda.org/conda-forge/gawk

Reply:
Yep. Just spotted that :+1: :smile:

Reply:
Fixed.